src/Account.sol

```diff
@@ -845,6 +919,11 @@ contract Account is IAccount, Auth, OpsReady {
         ConditionalOrder memory conditionalOrder =
```

Comment:
**Gas opt:** Not advocating for this now, as it would lead to more changes required to the code. But something that could be done for the next upgrade, where it can go through audit. Essentially in the `executeConditionalOrder` we read `conditionalOrder` from storage, and again in this function. It could be reduced to one read from storage.